analytics/analytics/settings.py

```diff
@@ -67,3 +67,5 @@
 
 GITLAB_ENDPOINT: str = os.environ["GITLAB_ENDPOINT"]
 GITLAB_TOKEN: str = os.environ["GITLAB_TOKEN"]
+
+SPACK_PROMETHEUS_ENDPOINT: str = os.environ.get("SPACK_PROMETHEUS_ENDPOINT")
```

Comment:
Should this use `.get` or require the endpoint be set?

Reply:
[cd8879d](https://github.com/spack/spack-infrastructure/pull/723/commits/cd8879d15f3e0990b842a7de2808cafced7e3f47)